include/veriblock/rewards/poprewards_params.hpp

```diff
@@ -0,0 +1,64 @@
+#ifndef ALT_INTEGRATION_INCLUDE_VERIBLOCK_REWARDS_POPREWARDS_PARAMS_HPP_
+#define ALT_INTEGRATION_INCLUDE_VERIBLOCK_REWARDS_POPREWARDS_PARAMS_HPP_
+
+#include <vector>
+
+namespace altintegration {
+
+// assume 8 decimals for all parameters
+static const uint32_t rewardsDecimalsMult = 100000000;
```

Comment:
maybe create a type, like `BigDecimal(2.1f) == 2.1f * rewardsDecomalsMult`?
So it internally uses int64_t for storage, but has interface functions to convert from float/double/integers